style.css

```diff
@@ -0,0 +1,60 @@
+.nav-bar {
+    display: flex;
+    background-color:#34b830;
+}
+
+.nav-menu {
+    margin: auto;
+}
+
+.nav-bar ul {
+    list-style-type: none;
+    margin: 0;
+    padding: 0;
+}
+
+li {
```

Comment:
avoid using tag selectors this are rarely used and usually we are using them to overwrite some default browser styling all over the project

Reply:
check entire file